src/examples/sqlite/src/backend/schema/track/entity.ts

```diff
@@ -0,0 +1,53 @@
+import {
+	Field,
+	GraphQLEntity,
+	ID,
+	ObjectType,
+	RelationshipField,
+	SummaryField,
+} from '@exogee/graphweaver';
+import { Album } from '../album';
+import { Genre } from '../genre';
+import { InvoiceLine } from '../invoice-line';
+import { MediaType } from '../media-type';
+import { Playlist } from '../playlist';
+import { Track as OrmTrack } from '../../entities';
+
+@ObjectType('Track')
+export class Track extends GraphQLEntity<OrmTrack> {
+	public dataEntity!: OrmTrack;
+
+	@Field(() => ID)
+	id!: number;
+
+	@SummaryField()
+	@Field(() => String)
+	name!: string;
+
+	@RelationshipField<Track>(() => Album, { id: (entity) => entity.album?.id, nullable: true })
+	album?: Album;
+
+	@RelationshipField<Track>(() => MediaType, { id: (entity) => entity.mediaType?.id })
+	mediaType!: MediaType;
+
+	@RelationshipField<Track>(() => Genre, { id: (entity) => entity.genre?.id, nullable: true })
+	genre?: Genre;
+
+	@Field(() => String, { nullable: true })
+	composer?: string;
+
+	@Field(() => Number)
+	milliseconds!: number;
+
+	@Field(() => Number, { nullable: true })
+	bytes?: number;
+
+	@Field(() => String)
+	unitPrice!: string;
```

Comment:
Same here, we should probably put a comment here about why prices are in strings or change it so it's a number type.